tests/Integration/Behaviour/Features/Scenario/Attribute/list_attribute_groups.feature

```diff
@@ -0,0 +1,67 @@
+@reset-database-before-feature
+@list-attribute-group
+Feature: Attribute Group
+  PrestaShop allows BO users to list attribute groups
+  As a BO user
+  I should be able to list attribute groups
+
+  Background:
+    Given language with iso code "en" is the default one
+    And attribute group "size" named "Size" in en language exists
+    And attribute group "color" named "Color" in en language exists
+    And attribute group "dimension" named "Dimension" in en language exists
+    And attribute group "paper_type" named "Paper Type" in en language exists
+    And attribute "s" named "S" in en language exists
+    And attribute "m" named "M" in en language exists
+    And attribute "l" named "L" in en language exists
+    And attribute "xl" named "XL" in en language exists
+    And attribute "grey" named "Grey" in en language exists
+    And attribute "taupe" named "Taupe" in en language exists
+    And attribute "beige" named "Beige" in en language exists
+    And attribute "white" named "White" in en language exists
+    And attribute "off_white" named "Off White" in en language exists
+    And attribute "red" named "Red" in en language exists
+    And attribute "black" named "Black" in en language exists
+    And attribute "camel" named "Camel" in en language exists
+    And attribute "orange" named "Orange" in en language exists
+    And attribute "blue" named "Blue" in en language exists
+    And attribute "green" named "Green" in en language exists
+    And attribute "yellow" named "Yellow" in en language exists
+    And attribute "brown" named "Brown" in en language exists
+    And attribute "pink" named "Pink" in en language exists
+
+  # NOTE: we don't initialize specific data since we already have the core features
+  Scenario: List attribute groups
+    When I list all attribute groups I should get following results:
+      | name[en-US] | public_name[en-US] | is_color_group | group_type | position | reference  |
+      | Size        | Size               | false          | select     | 0        | size       |
+      | Color       | Color              | true           | color      | 1        | color      |
+      | Dimension   | Dimension          | false          | select     | 2        | dimension  |
+      | Paper Type  | Paper Type         | false          | select     | 3        | paper_type |
+
+  Scenario: List attribute groups with attributes
+    When I list all attribute groups, the group "size" should have the following attributes:
+      | name[en-US] | color | position | reference |
+      | S           |       | 0        | s         |
+      | M           |       | 1        | m         |
+      | L           |       | 2        | l         |
+      | XL          |       | 3        | xl        |
+
+  Scenario: List attribute groups with attributes
+    When I list all attribute groups, the group "color" should have the following attributes:
```

Comment:
Once above mentioned action is adjusted, we can just remove the first and second scenario and instead cover it here in context.
```
Given I see a following list of attributes:
...
And I see following list of attribute groups with attributes:
...
Then the group "color" should have the following attributes:
...
```